README.md

```diff
@@ -0,0 +1,41 @@
+OpenShift Installer OpenStack Dev Scripts
+=========================================
+
+# Pre-requisites
+
+- CentOS 7
+- ideally on a bare metal host
+- user with sudo access
+
+# Instructions
+
+## 1) Create local config
+
+Copy an existing `user_config.sh` script and set values appropriate for your local environment.
+
+## 2) Run the scripts in order
+
+- `CONFIG="user_config.sh" ./01_install_requirements.sh`
+- `CONFIG="user_config.sh" ./02_run_all_in_one.sh`
+- `CONFIG="user_config.sh" ./03_configure_undercloud.sh`
+- `CONFIG="user_config.sh" ./04_ocp_repo_sync.sh`
+
+At step 5, you can run a minimal version that just builds an installer, or another version that sets up a local docker registry helpful if you are going to be making changes to the OpenStack machine actuator.
+
+- `CONFIG="user_config.sh" ./05_build_ocp_installer_minimal.sh`
```

Comment:
we could have this script called `build_ocp_installer` and the other one `05_build_ocp_depdendencies` ?

Reply:
Sure, good idea